micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyServerThreadPoolMetrics.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.jetty;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.binder.MeterBinder;
+
+public class JettyServerThreadPoolMetrics implements MeterBinder {
+
+    private final InstrumentedQueuedThreadPool threadPool;
+    private final Iterable<Tag> tags;
+
+    public JettyServerThreadPoolMetrics(InstrumentedQueuedThreadPool threadPool, Iterable<Tag> tags) {
+        this.threadPool = threadPool;
+        this.tags = tags;
+    }
+
+    @Override
+    public void bindTo(MeterRegistry registry) {
+        Gauge.builder("jetty.threads.min", threadPool, InstrumentedQueuedThreadPool::getMinThreads)
+             .tags(tags)
+             .description("The number of min threads")
+             .register(registry);
+        Gauge.builder("jetty.threads.max", threadPool, InstrumentedQueuedThreadPool::getMaxThreads)
+             .tags(tags)
+             .description("The number of max threads")
+             .register(registry);
+        Gauge.builder("jetty.threads.live", threadPool, InstrumentedQueuedThreadPool::getThreads)
```

Comment:
How does this compare to the Tomcat equivalent? Wondering if `active` is a better name or if it differs from busy? Or is it merely `idle + busy`?

Reply:
I decided the name `live` with reference to JvmThreadMetrics.
https://github.com/micrometer-metrics/micrometer/blob/v1.0.3/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmThreadMetrics.java#L57

For the meaning of the metric please refer to the Jetty's source.
https://github.com/eclipse/jetty.project/blob/jetty-9.4.8.v20171121/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java#L428-L455